### PR TITLE
fix: improving the implementation of digital filter to make it easier…

### PR DESCRIPTION
**Pull request by jwist**
… to maintain in the future

### CODE REVIEW

File: src/data/data1d/filter1d/fft.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ function generateXAxis(datum1D) {
   const info = datum1D.info;
   const baseFrequency = parseFloat(info.bf1);
   const spectralFrequency = parseFloat(info.sfo1);
-  const spectralWidth = parseFloat(info.sw);
+  const spectralWidth = parseFloat(info['spectral width']);
```
Comment (lpatiny) [inline src/data/data1d/filter1d/fft.js:53]: All the property names in info should be camelCase. Please rename it to spectralWidth

File: src/data/data1d/filter1d/lineBroadening.js [hunk 2]
```diff
@@ -12,9 +12,10 @@ export function apply(datum1D, value) {
     throw new Error('lineBroadening not applicable on this data');
   }
 
+  let grpdly = datum1D.info['digital filter'];
```
Comment (lpatiny) [inline src/data/data1d/filter1d/lineBroadening.js:15]: Also camelCase

File: src/data/utilities/getInfoFromMetaData.js [hunk 3]
```diff
@@ -24,17 +25,20 @@ export function getInfoFromMetaData(metaData) {
   maybeAdd(info, 'probe', metaData.$PROBHD);
   maybeAdd(info, 'bf1', metaData.$BF1);
   maybeAdd(info, 'sfo1', metaData.$SFO1);
-  maybeAdd(info, 'sw', metaData.$SW);
-  maybeAdd(info, 'numberOfPoints', metaData.$TD);
-  maybeAdd(info, 'dspfvs', metaData.$DSPFVS);
-  maybeAdd(info, 'decim', metaData.$DECIM);
-  maybeAdd(info, 'grpdly', metaData.$GRPDLY);
+  maybeAdd(info, 'spectral width', metaData.$SW);
```
Comment (lpatiny) [inline src/data/utilities/getInfoFromMetaData.js:28]: Also both camelCase

File: src/data/data1d/filter1d/lineBroadening.js (hunk 2 above)
Comment (jwist) [reply src/data/data1d/filter1d/lineBroadening.js]: I made the modifications

File: src/data/utilities/getInfoFromMetaData.js (hunk 3 above)
Comment (jwist) [reply src/data/utilities/getInfoFromMetaData.js]: I made the modifications

**Pull request merged**